deploy-manage/deploy/cloud-on-k8s/pod-disruption-budget.md

```diff
@@ -12,9 +12,45 @@ products:
 
 A [Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
 
-ECK manages a default PDB per {{es}} resource. It allows one {{es}} Pod to be taken down, as long as the cluster has a `green` health. Single-node clusters are not considered highly available and can always be disrupted.
+{{eck}} manages either a single default PDB, or multiple PDBs per {{es}} resource depending on the license level of the ECK installation.
 
-In the {{es}} specification, you can change the default behavior as follows:
+:::{note}
+In {{eck}} 3.1 and earlier, all clusters follow the [non-enterprise behavior](#non-enterprise-licensed-customers), regardless of license type.
+:::
+
+## Enterprise licensed customers
+```{applies_to}
+deployment:
+  eck: ga 3.2
+```
+
+In Elasticsearch clusters managed by ECK and licensed with an enterprise license, a separate PDB is created for each type of nodeSet defined in the manifest allowing upgrade or maintenance operations to be more quickly executed. Each PDB permits one Elasticsearch Pod per nodeSet to be disrupted at a time, provided the cluster maintains the health status described in the following table:
+
+| Role | Cluster health required | Notes |
+|------|------------------------|--------|
+| master | Yellow |  |
+| data | Green | All Data roles are grouped together into a single PDB, except for data_frozen. |
+| data_frozen | Yellow | Since frozen data tier nodes only host partially mounted indices backed by searchable snapshots additional disruptions are allowed. |
+| ingest | Yellow |  |
+| ml | Yellow |  |
+| coordinating | Yellow |  |
+| transform | Yellow |  |
+| remote_cluster_client | Yellow |  |
+
+Single-node clusters are not considered highly available and can always be disrupted regardless of license type.
+
+## Non-enterprise licensed customers
+:::{note}
+In {{eck}} 3.1 and earlier, all clusters follow this behavior regardless of license type.
+:::
+
+In {{eck}} clusters that do not have an enterprise license, one {{es}} Pod can be taken down at a time, as long as the cluster has a health status of `green`. Single-node clusters are not considered highly available and can always be disrupted.
+
+### Overriding the default behavior
```

Comment:
we still need to fix these heading levels per this comment https://github.com/elastic/docs-content/pull/2361#discussion_r2248711244

easiest way is to make this an H2 and its child an H3. alternative is outlined in the linked thread

Reply:
+1